src/main/java/seedu/mindmymoney/command/DeleteCommand.java

```diff
@@ -77,10 +78,10 @@ public void deleteExpenditure() throws MindMyMoneyException {
                 throw new MindMyMoneyException("Please input a valid index");
             } else {
                 assert positionToDelete >= 0 : "Index should always be >= 0";
-                System.out.println("I have removed "
+                System.out.println(PrintStrings.LINE + "I have removed "
```

Comment:
not sure if we shld use the line here, cos my add comand also dont have

Reply:
resolved